gateway/src/apicast/policy/scope_check/scope_check.lua

```diff
@@ -0,0 +1,105 @@
+local policy = require('apicast.policy')
+local _M = policy.new('Scope Check Policy')
```

Comment:
As @nmasse-itix pointed out, would be good to name this policy as Keycloak/RH-SSO specific if it only supports Keycloak/RH-SSO.

Reply:
Changed.